tip-wlan/charts/cassandra/templates/tests/test-cassandra-record-insert-del.yaml

```diff
@@ -12,6 +12,7 @@ spec:
   containers:
   - name: {{ include "common.name" . }}-test-cassandra-basic
     image: {{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}
+    imagePullPolicy: Always
```

Comment:
Instead of hardcoding in this and below yamls, can we use `imagePullPolicy: {{ .Values.global.pullPolicy | default .Values.pullPolicy }}` and let it be always over-ridden in the override-file at global or chart level.

Reply:
Wouldn't `imagePullPolicy: {{ .Values.pullPolicy | default .Values.global.pullPolicy }}` make more sense? Usually the chart's configuration has precedence over the global config, hasn't it?

Reply:
Yeah... that's fine. Main point is to not hardcode it in the Charts and let the chart level or global version decide it.